bot/converters.py

```diff
@@ -128,6 +134,44 @@ async def convert(self, ctx: Context, list_type: str) -> str:
         return list_type
 
 
+class Extension(Converter):
```

Comment:
I suppose it's okay to move these over for consistency's sake, but we should eventually have a discussion on this. I don't think it makes sense to define converters here which are unlikely to be used in multiple places. For example, `Duration` is generic and is indeed used in many places. But `OffTopicName`? That isn't used in more than one place nor will it likely ever be. We don't move cog-specific utlities to the `utils` directory, so why move such converters?

Reply:
I have wondered about this before; with this PR I decided to move them so the type reassignments are all in one place. I agree that it'd make more sense if they were in the modules of the cogs that use them though